_datademos/proof-how-to.md

```diff
@@ -16,67 +16,26 @@ This guide is intended to describe how you can run PROOF, catering to varying le
 
--   **Robust Workflow Management**: Cromwell offers a robust engine for managing complex workflows, ensuring efficient execution of tasks and streamlined workflow orchestration.
+- Run **multiple workflows** simultaneously from a single PROOF server
+- **Track** the status of all your workflows and tasks in one place
+- Use **cached results** to skip redundant computations
+- **Query** your workflow history for outputs and metadata
 
--   **Scalability**: With its ability to handle large-scale workflows, Cromwell accommodates projects of varying sizes, from small-scale analyses to large-scale data processing pipelines.
-
--   **Comprehensive Job Monitoring**: It provides comprehensive job monitoring and metadata tracking, enabling users to closely monitor workflow progress, identify bottlenecks, and troubleshoot issues effectively.
-
--   **Community Support and Documentation**: Cromwell benefits from a supportive community and extensive documentation, offering users access to resources, tutorials, and community-driven solutions to common challenges.
-
--   **Open Source and Customizable**: Being open-source, Cromwell allows for customization and adaptation to specific workflow requirements, empowering users to tailor workflows to their unique needs and preferences.
-
-
-*How should we use Cromwell?*
-
-In general, Cromwell works best when run in **server mode**, which means that users start a **Cromwell server** as a job on our local SLURM cluster that can connect to a **database** specifically for Cromwell workflow tracking.
-  
-This Cromwell server job then behaves as the workflow **coordinator** for that **user**, allowing a user to send workflow instructions for **multiple workflows** running simultaneously.
-
-The Cromwell server will then **parse** these workflow instructions, find and copy the relevant input files, **send** the tasks to the cluster (Gizmo) to be processed, **coordinate** the results of those tasks and **record** all of the metadata about what is happening in its database.
-
-This means that individual users can:
-
- - Run **multiple independent workflows** at the same time using **one Cromwell server**
-
- - Use **cached results** when identical to the current task
-
- - **Track** the status of workflows and tasks
- 
- - **Customize** the locations of input data, intermediate data, and workflow outputs into data storage resources appropriate to the data type (re: cost, backup, and accessibility)
-
- - **Query** the Cromwell database for information about workflows run in the past, including where their workflow outputs were saved or a variety of other workflow and task level metadata.
- 
-> Quick note: the Cromwell server is referred to as a PROOF server in these instructions. PROOF handles setting up the Cromwell server for you.
+> Quick note: Throughout this guide, "PROOF server" and "Cromwell server" refer to the same thing - PROOF just handles all the Cromwell setup for you.
 
 ## Using PROOF
```

Comment:
Under this section it says to use the Marconi network, is that still true? It should be Research Staff now, right? Line 51